test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -168,12 +217,15 @@ test
     });
     await challengeEmailPageObject.clickEnterCodeLink();
 
-    const pageTitle = challengeEmailPageObject.getPageTitle();
+    const pageTitle = challengeEmailPageObject.getFormTitle();
     const saveBtnText = challengeEmailPageObject.getSaveButtonLabel();
-    await t.expect(saveBtnText).contains('Verify');
     await t.expect(pageTitle).contains('Verify with your email');
+    await t.expect(saveBtnText).notContains('Send me an email');
+    await t.expect(saveBtnText).contains('Verify');
```

Comment:
nit: instead of 2 expects, `notContains` followed by `contains`, we could do `eql` once and check that the text is correct (not sure why we don't do it already, maybe there is something I am missing)